backend/groth16/bls12-377/prove.go

```diff
@@ -37,9 +38,10 @@ import (
 // with a valid statement and a VerifyingKey
 // Notation follows Figure 4. in DIZK paper https://eprint.iacr.org/2018/691.pdf
 type Proof struct {
-	Ar, Krs                   curve.G1Affine
-	Bs                        curve.G2Affine
-	Commitment, CommitmentPok curve.G1Affine
+	Ar, Krs       curve.G1Affine
+	Bs            curve.G2Affine
+	CommitmentPok curve.G1Affine
```

Comment:
can you add a comment here to point to the right resource or TLDR; what is CommitmentPok and Commitments?